src/bin/stratisd.rs

```diff
@@ -75,15 +69,21 @@ fn main() {
         .get_matches();
 
     if matches.is_present("debug") {
-        unsafe { DEBUG = true }
-    };
+        let mut builder = LogBuilder::new();
```

Comment:
The format is the same as the default that we get when we call env_logger::init().

Can you explain your objection to parse()?  It is more efficent to call filter()?